src/measure_theory/measurable_space_def.lean

```diff
@@ -377,10 +377,10 @@ begin
   { exact measurable_set_generate_from ht, },
 end
 
-@[simp] lemma generate_from_singleton_empty : generate_from {∅} = (⊥ : measurable_space α) :=
+lemma generate_from_singleton_empty : generate_from {∅} = (⊥ : measurable_space α) :=
```

Comment:
Can you explain why you removed this `@[simp]` and the one of `generate_from_singleton_univ`?

Reply:
They are now provable by simp